zerver/openapi/zulip.yaml

```diff
@@ -1206,25 +1266,1581 @@ paths:
+                                      type: integer
+                                      description: |
+                                        The id of the deleted attachment.
+                                upload_space_used:
+                                  type: integer
+                                  description: |
+                                    The total size of all files uploaded by in the organization,
+                                    in bytes.
+                              example:
+                                {
+                                  "type": "attachment",
+                                  "op": "remove",
+                                  "attachment": {"id": 1},
+                                  "upload_space_used": 0,
+                                  "id": 0,
+                                }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent when a submessage is added to a message.
+                              properties:
+                                id:
+                                  type: integer
+                                type:
+                                  type: string
+                                  enum:
+                                    - submessage
+                                msg_type:
+                                  type: string
+                                  description: |
+                                    The type of the message.
+                                content:
+                                  type: string
+                                  description: |
+                                    The content of the submessage.
+                                message_id:
+                                  type: integer
+                                  description: |
+                                    The id of the message to which the submessage has been added.
+                                sender_id:
+                                  type: integer
+                                  description: |
+                                    The id of the user who sent the message.
+                                submessage_id:
+                                  type: integer
+                                  description: |
+                                    The id of the submessage.
+                              example:
+                                {
+                                  "type": "submessage",
+                                  "msg_type": "whatever",
+                                  "message_id": 34,
+                                  "submessage_id": 1,
+                                  "sender_id": 8,
+                                  "content": '"stuff"',
+                                  "id": 0,
+                                }
```

Comment:
This example looks extremely artificial.  Can you capture a real payload from watching the network tab when using `/poll`?

Reply:
Done